src/Glpi/Inventory/Conf.php

```diff
@@ -384,26 +384,63 @@ public function showConfigForm()
             echo '</div>';
             echo "</td>";
             echo "</tr>";
-
             echo "<tr class='tab_bg_1'>";
             echo "<td>";
             echo "<label for='auth'>" . __s('Authorization header') . "</label>";
             echo "</td>";
             echo "<td>";
             Dropdown::showFromArray('auth_required', [
                 'none' => __('None'),
-                'client_credentials' => __('OAuth - Client credentials')
+                'client_credentials' => __('OAuth - Client credentials'),
+                'basic_authentification' => __('Basic Authentification'),
```

Comment:
Can you move `client_credentials` and `basic_authentification` as `const`

```php
const CLIENT_CREDENTIALS = 'client_credentials';
const BASIC_AUTH = 'basic_auth';
```

allows us (among other things) to change only one place (the day we want to update the value of the constant).